patches/tock/02-usb.patch

```diff
@@ -215,8 +215,8 @@ index ebee8704b..6a2f8715b 100644
              kernel::ipc::DRIVER_NUM => f(Some(&self.ipc)),
              _ => f(None),
          }
-@@ -371,6 +388,21 @@ pub unsafe fn reset_handler() {
-         )
+@@ -377,6 +394,21 @@ pub unsafe fn reset_handler() {
+             .expect("no deferred call slot available for nvmc"),
```

Comment:
Why is USB impacted if we don't change the reference commit for the kernel?

Reply:
This is just a context line. The `+` is actually part of the file not part of the diff. It's like for the line numbers that get updated for the patch to be applied perfectly.

Reply:
I still read `no deferred call slot available for nvmc` for a patch about USB.
I doubt this was there before ;)

Reply:
Yes it wasn't there before this PR. The usb patch is applied after the boards are copied and after the persistent storage patch. So with this PR, when applying the usb patch, this line is actually present.

Reply:
Gotcha.